service/controller/v14patch2/resource/deployment/master_deployment.go

```diff
@@ -76,10 +76,13 @@ func newMasterDeployments(customObject v1alpha1.KVMConfig) ([]*extensionsv1.Depl
 					key.VersionBundleVersionAnnotation: key.VersionBundleVersion(customObject),
 				},
 				Labels: map[string]string{
-					"cluster":  key.ClusterID(customObject),
-					"customer": key.ClusterCustomer(customObject),
-					"app":      key.MasterID,
-					"node":     masterNode.ID,
```

Comment:
Current labels stay for now even though some are deprecated. Added new ones though.